syntax = "proto3";
package proto;

//python -m grpc_tools.protoc -I. --python_out=. --grpc_python_out=. ./iris.proto

service PredictionTask {
  rpc LaunchTask(TaskRequest) returns (TaskResponse){}
}
message TaskRequest {
  string sepal_length = 1;
  string  sepal_width = 2;
  string petal_length = 3;
  string petal_width = 4;
}
message TaskResponse{
  message Result {
    string class = 1;
    string probability = 2;
  }
  repeated Result result = 1;
}

